apps/site/assets/css/_amenity.scss

```diff
@@ -15,3 +15,47 @@
     margin: .65rem;
   }
 }
+
+.access-amenities-header {
+  background: $brand-primary-darkest;
+  color: $white;
+  height: 3.75rem;
+}
+
+tr:nth-child(2n+3) {
+  background-color: $gray-lightest;
+}
+
+.access-amenities-row {
+  height: 5rem;
+}
+
+.status {
+  padding-right: 3.125rem;
+}
+
+.access-amenities-name {
+  padding-left: 1rem;
+}
```

Comment:
non-blocking: There is a `_spacing.scss` file that contains the `padding-left: 1rem;` as a helper class already. 
 The `padding-right: 3.125rem` could be added to it as well.